mesonbuild/wrap/wrap.py

```diff
@@ -554,7 +563,6 @@ def get_file_internal(self, what: str) -> str:
             self.download(what, cache_path)
             return cache_path
         else:
-            from ..interpreterbase import FeatureNew
             FeatureNew(f'Local wrap patch files without {what}_url', '0.55.0').use(self.current_subproject)
```

Comment:
You forgot to remove that line. You can also remove self.current_subproject that should be unused now.